components/widgets/badges/badge.scss

```diff
@@ -6,7 +6,7 @@
     padding: 1px 4px;
     margin: 2px 10px 0 -4px;
 
-    &.BotBadge {
+    &.bot-badge {
```

Comment:
Do we have any documentation on how to structure the CSS for the widget library? It would be good to have something to reference for these things since they clash with our existing naming conventions